coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -12,6 +12,7 @@ class DocumentationComment:
     inside source-code, like position etc.
     """
     Parameter = namedtuple('Parameter', 'name, desc')
+    ExceptionValue = namedtuple('ExceptionValue', 'name, desc')
```

Comment:
oh wow totally forgot about that

let's keep it then? do you have a better suggestion?

`ReturnValue` has a value, so it made sense.

If your suggestion also involves changing `Parameter`, then that's fine.